File: gui/src/main/java/com/woohyman/gui/base/EmulatorView.java
package com.woohyman.gui.base;

import android.view.View;

import com.woohyman.keyboard.base.ViewPort;

interface EmulatorView {
    void onPause();

    void onResume();

    void setQuality(int quality);

    ViewPort getViewPort();

    View asView();
}

File: gui/src/main/java/com/woohyman/gui/base/EmulatorView.kt
package com.woohyman.gui.base

import android.view.View
import com.woohyman.keyboard.base.ViewPort

internal interface EmulatorView {
    fun onPause()
    fun onResume()
    fun setQuality(quality: Int)
    val viewPort: ViewPort?
    fun asView(): View
}